grails-app/controllers/org/broadinstitute/orsp/api/ExceptionHandler.groovy

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.orsp.api
+
+import grails.converters.JSON
+import groovy.util.logging.Slf4j
+
+
+@Slf4j
+trait ExceptionHandler {
+
+    def handleException(Exception e, Integer statusCode) {
```

Comment:
I do think this is OK in this form, but it could be a bit smarter and more dev-friendly if this handles something similar to https://docs.oracle.com/javaee/7/api/javax/ws/rs/ClientErrorException.html and then you could get the status code from the exception itself. There could also be a handler for any exception case similar to this one. Pretty sure that library (javax.ws.rs-api) is not in this project - we have `org.springframework.web.client.HttpClientErrorException` which is much less helpful, ugh. 

Looking through the code, there are 3 cases, 400, 404, and 500. Might be easier to just have `handleNotFound`, `handleBadRequest`, and `handleException` that takes an optional string for the message.